rune/libenclave/intelsgx/arch.go

```diff
@@ -115,12 +115,20 @@ type Quote struct {
 const (
 	QuoteSignatureTypeUnlinkable = iota
 	QuoteSignatureTypeLinkable
+	InvalidQuoteSignatureType
 )
 
 const (
 	QuoteVersion = 2
 )
 
+// RA Type
+const (
+	DCAP = iota
+	EPID
+	InvalidRaType
+)
```

Comment:
And move this enum to libenclave/attestation/sgx/attest.go, because this enum is not part of architecture.